### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by urso** [xataio/pgzx#57]
The readme is getting quite big and it is becoming difficult for users to find the right information or we didn't yet document things properly. To improve the DX and especially getting started experience I would like to split the readme out into a proper documentation.

TODO: sections/structure?

**Comment by aleclarson** [comment]
I see the docs folder. Is that accessible anywhere?

**Comment by urso** [comment]
Yes, the docs are published via github pages: https://xataio.github.io/pgzx/#test.pgzx

Looks like something changed in docs building last time I checked. Everything is now reported under the `test` namespace. But these are generated docs only. These need some more work as well, but I'm thinking to provide some more how-to like docs.
